metadata-io/src/main/java/com/linkedin/metadata/systemmetadata/ESSystemMetadataDAO.java

```diff
@@ -43,6 +45,24 @@ public class ESSystemMetadataDAO {
   private final ESBulkProcessor bulkProcessor;
   private final int numRetries;
 
+  /**
+   * Gets the status of a Task running in ElasticSearch
+   * @param taskId the task ID to get the status of
+   */
+  public Optional<GetTaskResponse> getTaskStatus(@Nonnull String nodeId, long taskId) {
+    final GetTaskRequest taskRequest = new GetTaskRequest(
+        nodeId,
+        taskId
+    );
+    try {
+      return client.tasks().get(taskRequest, RequestOptions.DEFAULT);
+    } catch (IOException e) {
+      log.error(String.format("ERROR: Failed to get task status for %s:%d. See stacktrace for a more detailed error:", nodeId, taskId));
+      e.printStackTrace();
```

Comment:
```suggestion
     
```

Remove printStackTrace

Reply:
You can add the `e` to the log.error("message", e) to get this in the log